in_toto/model.go

```diff
@@ -68,6 +68,27 @@ type Layout struct {
   Readme string `json:"readme"`
 }
 
+// Go does not allow to pass `[]T` (slice with certain type) to a function
+// the accepts `[]interface{}` (slice with generic type)
```

Comment:
s/the/that/

Reply:
a685b474ff64e059aabbc1feb125c899305db28c